### DISCUSSION TITLE
Ssl certs not being read and falsely flags as invalid

### DISCUSSION THREAD

**Discussion by therealMrFunGuy** [phantom/docs#242] [Help & Support]
### Summary

This issue has been progressively getting worse the more updates that come out.

https://Rjctdlabs.xyz is our platform

If you load it up once in mobile phantom, it might work, but if you try again, it won't until you force quit the app and clear cache and cookies...

This is affecting android and ios mobile phantom wallets

I've noticed zero issues with solflare, mobile chrome, and all pc/Mac, it's only phantom we have this issue with 😅

### Steps to Reproduce

Just have to open phantom mobile and try to go to rjctdlabs.xyz. if it works, try refreshing. It that works, please open a new tab and try to access the site

![Screenshot_20240530_151031_Phantom](https://github.com/phantom/docs/assets/88522042/3fee9b90-5d37-4149-aabf-b1722d43a08b)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi @therealMrFunGuy,

I will try more later today, but I have yet to be able to reproduce this using your instructions. Are you still seeing it?

**Reply by therealMrFunGuy** [reply]
This has been an issue getting worse for months, I have literally 50+ reports of this each day...

Have you found anything yet?

**Comment by therealMrFunGuy** [comment]
Yes, consistently, to the point where we have users opening tickets daily...

If you are on mobile, only way for me to load after a while is to force
quit and clear cache

For some that doesn't work either and they have to use solflare

**Reply by adamdelphantom** [reply]
I still haven't been able to reproduce on iOS or android.

If you have a device that you can reliably reproduce on I would recommend using this to see if the console provides any errors that can help debug.

https://docs.phantom.app/developer-powertools/mobile-web-debugging

**Reply by therealMrFunGuy** [reply]
yo fam, can you try again?

This is affecting literally hundreds of users every day... I'm not a developer and don't understand how to debug your software...
